### PR TITLE
Minor. Update api doc about source regarding json response for auto_responses, signatures and from_name. RD-23635

### PR DESCRIPTION
**Pull request by onima**
refactor_api_step_1

This  will displayed on 4 endpoints for source (2 GET and 2 PUT)

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -8365,7 +8365,7 @@ components:
           description: Only on Engage Messaging sources. Boolean describing if the source is enabled on the web or not.
           type: boolean
         auto_response_triggers:
-          description: Only on Email sources. Automatic messages.
+          description: Only on sources which support auto response triggerable. Automatic messages.
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:8368]: I'm not sure if I should list instead all sources that support auto response triggerable (I think it's something like 8 sources)

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -8376,30 +8376,33 @@ components:
...
+          description: Signatures.
           type: array
           items:
             properties:
               language:
                 type: string
               body:
                 type: string
+        from_name:
+          description: Custom Sender
```
Comment (onima) [inline specs/engage-digital_openapi3.yaml:8404]: Couldn't find something more meaningful :D

File: specs/engage-digital_openapi3.yaml (hunk 1 above)
Comment (kalilz4485) [reply specs/engage-digital_openapi3.yaml:8368]: lets not list them but better use the feature name e.g. "Only on sources which supports Automatic Messages"

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -8376,30 +8376,33 @@ components:
               body:
                 type: string
         auto_response_trigger_enabled:
-          description: Only on Email sources. Boolean describing if answers are enabled to first messages and/or to messages outside business hours.
+          description: Only on sources which support auto response triggerable. Boolean describing if answers are enabled to first messages and/or to messages outside business hours.
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:8379]: lets put the real description before the "Only on" part

File: specs/engage-digital_openapi3.yaml (hunk 2 above)
Comment (kalilz4485) [reply specs/engage-digital_openapi3.yaml:8404]: "Custom sender name. Only on sources which supports a custom sender name."

**Comment by onima** [comment]
It should be better now :)

**Comment by onima** [comment]
Ping @kalilz4485 :)

**Comment by kalilz4485** [comment]
LGTM

**Pull request merged**